supertokens_python/querier.py

```diff
@@ -259,10 +259,17 @@ async def __send_request_helper(
                     + response.text  # type: ignore
                 )
 
+            res: Dict[str, Any] = {"_headers": dict(response.headers)}
+
+            if response.headers.get("content-type").startswith("text"):
+                res["_text"] = response.text
+
             try:
-                return response.json()
+                res.update(response.json())
             except JSONDecodeError:
-                return response.text
+                pass
+
+            return res
```

Comment:
I tried with creating a QuerierResponse class as well but that will require too many changes. Plus, we aren't actually using response.text anywhere. So made it part of the dict response.